scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -313,7 +347,7 @@ object Execution {
       new EvalCache {
         override protected[EvalCache] val messageQueue: LinkedBlockingQueue[EvalCache.FlowDefAction] = self.messageQueue
         override def start(): Unit = sys.error("Invalid to start child EvalCache")
-        override def finished(): Unit = sys.error("Invalid to finish child EvalCache")
+        override def finished(mode: Mode): Unit = sys.error("Invalid to finish child EvalCache")
```

Comment:
don't we need to override the `addFilesToCleanup` to forward back to the parent list? I think any files added to the child will not be deleted, right?